src/components/login-view/login-view.jsx

```diff
@@ -13,7 +16,7 @@ export const LoginView = ({ onLoggedIn }) => {
       Password: password,
```

Comment:
Even at this point your API expects this to be lower case, you can see that from here: https://github.com/allmyprogramming/movie_api/blob/14dd9c075086c77494496461bf045f4ee0a5d969/index.js#L111C10-L111C18. When test your application, it will fail because:-

```
const data = {
      Username: username,
      Password: password,
}
```
 and 
 
 ```
const data = {
      username: username,
      password: password,
}
```
are not the same things. Javascript is strict about variable names.